x/rollapp/module.go

```diff
@@ -168,5 +168,6 @@ func (am AppModule) GetHooks() []types.RollappHooks {
 // returns no validator updates.
```

Comment:
docstring should say a bit more about the behaviour

Reply:
done